plugins/scheduled_task/lib/processors/KGenericProcessor.php

```diff
@@ -48,6 +48,28 @@ protected function additionalActions($profile, $object, $tasksCompleted, $error)
 		//Nothing To Do in case of Generic.
 	}
 
+	protected function shouldProcessTask($object, $objectTask)
+	{
+		return $object instanceof KalturaBaseEntry && $objectTask->stopProcessingOnError
+			&& in_array($objectTask->type, $this->getSupportedTypesForProcess()) && $this->isPartnerSupportedForProcess($object->partnerId);
+	}
+
+	protected function getSupportedTypesForProcess()
+	{
+		$distributeTaskType = ScheduledTaskContentDistributionPlugin::getApiValue('Distribute');
```

Comment:
dont we have an enum for distribute

Reply:
not working - Interface 'IKalturaPluginEnum' not found